packaging_automation/common_tool_methods.py

```diff
@@ -32,10 +36,39 @@ def release_all_repos():
         repo.close()
 
 
+class PackageType(Enum):
+    deb = 1,
+    rpm = 2
+
+
+class GpgKeyType(Enum):
+    private = 1
+    public = 2
+
+
+BASE_PATH = pathlib2.Path(__file__).parents[1]
+
+
 def get_spec_file_name(project_name: str) -> str:
     return f"{project_name}.spec"
 
 
+def get_minor_project_version(project_version: str) -> str:
+    project_version_details = get_version_details(project_version)
+    return f'{project_version_details["major"]}.{project_version_details["minor"]}'
+
+
+def get_version_number(version: str, fancy: bool, fancy_release_count: int) -> str:
+    fancy_suffix = f"-{fancy_release_count}" if fancy else ""
+    return f"{version}{fancy_suffix}"
+
+
+def get_version_number_with_project_name(project_name: str, version: str, fancy: bool,
+                                         fancy_release_number: int) -> str:
+    fancy_suffix = f"-{fancy_release_number}" if fancy else ""
+    return f"{version}.{project_name}{fancy_suffix}"
```

Comment:
1) `_number` word doesn't sound much suitable for those func names, since we are indeed returning strings in both.

Can you come up with better names for those functions ?

Maybe `get_version_number` -> `process_version_string` & `get_version_number_with_project_name` -> `process_version_with_project_name`

2) `fancy_release_count` -> `fancy_release_number`